src/witness_generator/bit_constraints.rs

```diff
@@ -149,7 +142,11 @@ pub fn determine_global_constraints<'a>(
     }
 
     if fixed_data.verbose {
-        println!("Determined the following identities to be bit/range constraints:");
+        println!("Determined the following global bit constraints:");
+        for (name, con) in &known_constraints {
+            println!("  {name}: {con}");
+        }
+        println!("Determined the following identities to be purely bit/range constraints:");
```

Comment:
We should probably have some verbosity mode for these in the future

Reply:
You mean different verbosity levels?

Reply:
Or just a Boolean. Right now is this always printed?

Reply:
` if fixed_data.verbose {`

Reply:
https://docs.rs/env_logger/latest/env_logger/

This works well